### DISCUSSION TITLE
Azure PsRule, check if resource exists, if so validate that there is another resource with it as parent

### DISCUSSION THREAD

**Discussion by C0smin** [microsoft/PSRule#1392] [Q&A] [answered]
Hey guys, 

This has been bugging me for a while now. 
I want to check if this deployment has a storage account and if it has check if there is also a resource of type 'Microsoft.Security/advancedThreatProtectionSettings'. 
I found that you can do that with json arm templates using $PSRule.GetContent($TargetObject)[0].resources but cant' figure out how to do it for bicep.
Do you have some pointers?

Thanks,
Cosmin

**Comment by BernieWhite** [comment, marked as answer]
Duplicate question answered over here: https://github.com/Azure/PSRule.Rules.Azure/discussions/1969
